docs/finops-hub/template.md

```diff
@@ -60,12 +61,22 @@ Resources use the following naming convention: `<hubName>-<purpose>-<unique-suff
       > ℹ️ _In the future, we will use this container to stage external data outside of Cost Management._
     - `config` – Stores hub metadata and configuration settings. Files:
       - `settings.json` – Hub settings.
+      - `schema_ea.json` – Transforms for EA data.
+      - `schema_mca.json` – Transforms for MCA data.
 - `<hubName>-engine-<unique-suffix>` Data Factory instance
   - Pipelines:
+    - `msexports_backfill` – Triggers a series of monthly Cost Management exports (msexports_fill pipeline) to fill the dataset per the retention setting defined in settings.json
+    - `msexports_fill` – Creates and triggers Cost Management exports (both actual and amortized) for the selected scope and date range.
```

Comment:
When I look at all these names, I feel like they're not very understandable without reading the docs. Given they're internal, should we care?